std/exception.d

```diff
@@ -1642,7 +1645,8 @@ class ErrnoException : Exception
     this(string msg, int errno, string file = null, size_t line = 0) @safe
     {
         _errno = errno;
-        super(msg ~ " (" ~ errnoString(errno) ~ ")", file, line);
+        _errnoMsg = errnoString(errno);
```

Comment:
Why is the message eagerly allocated and stored as a field? I would lazily initialize it when the function is called.

Reply:
You're right, I misread the code. My bad!